cmake_modules/gmCommonSetup.cmake

```diff
@@ -32,8 +40,128 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
 
   cmake_policy ( SET CMP0011 NEW )
 
+  option(CLIENTTOOLS "Enable the building/inclusion of a Client Tools component." OFF)
+  option(PLATFORM "Enable the building/inclusion of a Platform component." ON)
```

Comment:
CLIENTTOOLS and PLATFORM are not needed.  During the build some of these tags are needed by the platform.